server.go

```diff
@@ -104,12 +104,12 @@ func (server *Server) searchHandler(w http.ResponseWriter, r *http.Request) {
 
 func (server *Server) flattenEntities(result *[]volkszaehler.Entity, entities []volkszaehler.Entity, parent string) {
 	for _, entity := range entities {
-		if entity.Type == "group" {
+                if parent != "" {
```

Comment:
formatting looks odd?

Reply:
I have planned to do two pull separate pull requests one for the group stuff and one for the formatting but failed with github. :-(

I have updated the comment.